specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,81 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. The Browser Task Manager is a window helpful for debugging that 
+displays the different processes associated with the current browser process and 
+what they're used for. For instance, it could denote that a particular process 
+is a renderer process and would show the different web pages rendered by that 
+process. In a chromium browser it can be opened by the end user by pressing 
+`Shift+Esc` or from the DevTools window's title bar's context menu entry 
+`Browser task manager`. For WebView2 applications, we block the 
+`Shift+Esc` shortcut, but the task manager can still be opened via the 
+`Browser task manager` entry.
+
+At the current time, it is expected that the user will close the Task 
+Manager manually, so we do not need to provide an API to close it.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending `CoreWebView2` to provide an `OpenTaskManagerWindow` 
+method. This method will open a new window containing the task manager. 
+If the task manager is already opened, this method will do nothing.
+
+# Examples
+## C++: Open Task Manager
+
+``` cpp
+wil::com_ptr<ICoreWebView2_5> m_webview;
+
+// This method could be called from a menu bar item, such as 
+// [Script -> Open Task Manager Window]. 
+void ScriptComponent::OpenTaskManagerWindow()
+{
+    CHECK_FAILURE(m_webview->OpenTaskManagerWindow());
```

Comment:
This will always AV because `m_webview` is never initialized. Presumably this should be something like

```cpp
    if (auto webview5 = m_webview.try_query<ICoreWebView2_5>(); webview5)
    {
        CHECK_FAILURE(webview5->OpenTaskManagerWindow());
    }
```

And also

```cpp
// Assume OpenTaskManagerWindow is available if the WebView supports ICoreWebView2_5.
bool ScriptComponent::ShouldShowOpenTaskManagerWindowMenuItem()
{
    return m_webview.try_query<ICoreWebView2_5>();
}
```

Reply:
Added initialization now as well as method for checking for support for adding the menu item.